pkg/controller/v1beta1/inferenceservice/reconcilers/deployment/deployment_reconciler.go

```diff
@@ -66,6 +67,7 @@ func createRawDeployment(componentMeta metav1.ObjectMeta,
 	deployment := &appsv1.Deployment{
 		ObjectMeta: componentMeta,
 		Spec: appsv1.DeploymentSpec{
+			Replicas: componentExt.Replicas,
```

Comment:
Do we need this line as `Replicas` is set in `setDeploymentReplicas`

Reply:
This is a good catch, as it can conflict with the autoscaler.

I changed the validating webhook. So, the `componentExt.Replicas` should come with `nil` value when using an autoscaler. But I think it is better to not have any risk of conflicts.

I removed this line.